modules/nlu/src/backend/api.ts

```diff
@@ -205,6 +219,27 @@ export default async (bp: typeof sdk, state: NLUState) => {
     const { botId, id } = req.params
     const botEngine = state.nluByBot[botId].engine1 as ScopedEngine
     await botEngine.storage.deleteEntity(id)
+
+    const intents = await botEngine.storage.getIntents()
+    const awaitsList = intents.map(async intent => {
```

Comment:
can you use `Promise.map` instead of  map + Promise .all

Reply:
Ok, done, ty.